documentation/colors.md

```diff
@@ -0,0 +1,27 @@
+# Colors support
+
+DNSControl is using colors on both its console and notification outputs, to
+denote changes, such as during preview or push events.
+
+This is on by default in all interactive prompts supporting colors. See below on
+how to bypass it.
+
+## Disable colors
+
+There are cases that we might need to disable colored output, such as in certain
+CI environments or particular notification mechanisms that don't support ANSI color
+codes.
+
+In order to do so, a global `--no-colors` command option is provided, which when
+set, will disable colors globally.
```

Comment:
Set to what value? Perhaps we can help a newcomer with a small example, by describing the `--no-colors=true` case here? The two `--no-colors=false` cases are described below (`(Force) Enable colors` & `Force colors in CI environments`).

```suggestion
set `--no-colors=true`, will disable colors globally.
```